bangazonapi/views/profile.py

```diff
@@ -233,17 +233,16 @@ def cart(self, request):
             """
 
             try:
-                open_order = Order.objects.get(customer=current_user)
-                print(open_order)
+                open_order = Order.objects.filter(customer=current_user).last()
```

Comment:
This technically will work but it's not obvious why it works (and I think it's causing the AssertionError in your comment). Since an open order is one where the payment_type is null, you can add that condition to the filter instead of using `.last()`

Reply:
I have added a filter for `payment_type=None`, but unfortunately when I run this code (`open_order = Order.objects.filter(customer=current_user, payment_type=None)`), I get an ValueError: `ValueError: Cannot assign "<QuerySet [<Order: Order object (11)>]>": "OrderProduct.order" must be a "Order" instance.` It does work if the `.last()` remains at the end of line 236 though.

Reply:
Added a filter for `payment_type=None` and reverted method to `.get()`.